static/scripts/main.js

```diff
@@ -3,7 +3,7 @@ var UnpinOffset = 400;
 window.onload = function () {
     // fix first page load anchor issue
     var url = window.location.hash;
-    var divid = url.split('#');
+    var divid = url.split('#!');
```

Comment:
this looks wrong
this code splits the _has_ itself to find a hash-in-hash that identifies an element on the page (like heading) to scroll to it